test/Datadog.Trace.ClrProfiler.Managed.Tests/AttributeTests.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="AttributeTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.Managed.Tests
+{
+    public class AttributeTests
+    {
+        /// <summary>
+        /// This test reports instances in the Datadog.Trace.ClrProfiler.Managed and Datadog.Trace
+        /// where attributes use named arguments that are not a built-in type. Using a custom type
+        /// (especially one from the Datadog.Trace assembly) as a named argument can fail with a
+        /// System.Reflection.CustomAttributeFormatException if the owning assembly cannot be found
+        /// in the default load context and other assembly resolve events are registered, in addition
+        /// to the ones added by the Datadog .NET Tracer
+        /// </summary>
+        [Fact]
+        public void AttributesInstantiationsOnlyUseBuiltinTypes()
+        {
+            List<string> invalidAttributeUsages = new();
+            var coreAssembly = typeof(object).Assembly;
```

Comment:
Did you meant to use something like `typeof(Tracer)` here? We want attributes from `Datadog.Trace.dll` and `Datadog.Trace.ClrProfiler.Managed.dll`, right?

Reply:
Oh, I guess not. I just noticed the `if (argumentType.Assembly != coreAssembly)` below. I still don't understand why we iterate over all types in `typeof(object).Assembly`, though.

Reply:
🤦🏼 Yep exactly, I meant to do `typeof(Tracer)`. I still need the core assembly to do the assembly comparison later, so I'll just add a small fix to get the Trace assembly. Fixed in 78269a33c2f22c5e875a6a6a227797795e259bc2